scripts/install.sh

```diff
@@ -3,8 +3,15 @@
 set -e
 set -o pipefail
 
+rustup_commandline="curl https://sh.rustup.rs -sSf | sh -s -- -y"
```

Comment:
Seems that `rustup_commandline` is better, `line` suffix has no meaning here.

We should use `RUSTUP_COMMAND` or keep it lowercased `rustup_command`, but then all other variables (script local) should be lowercased as well (like `HERE`, ...).